packages/notifi-react-card/README.md

```diff
@@ -1,45 +1,88 @@
 # `@notifi-network/notifi-react-card`
 
-> Configurable component for Notifi alerts
+A drop-in component that can get your dapp up and running with alert notifications in minutes.
 
-## Usage
+We currently support the following blockchains:
+
+- Solana
+- Ethereum
+- Polygon
+- Arbitrum
+- Binance
+- Aptos
+- Acala
+
+# Getting Started
+
+Fill out [this form](https://bit.ly/NotifiDappSetup) to receive a developer login.
+
+With a developer login, you can generate an API key and customize component configurations.
+
+&nbsp;
+
+# Installation
+
+**npm**
 
 ```
 npm install @notifi-network/notifi-react-card
 npm install --save-dev @notifi-network/notifi-core
 ```
 
-You can import the default stylesheet to get baseline styling.
-Be sure to get your DAPP ID here: https://bit.ly/NotifiDappSetup
+**yarn**
 
-## Video
+```
+yarn add @notifi-network/notifi-react-card
+yarn add --dev @notifi-network/notifi-core
+```
 
-Here is a [link](https://www.youtube.com/watch?v=LIKu-clf9bg) covering how to setup the React Card Config.
+Import the following CSS file into your component to get baseline styling:
 
-The video covers the following:
+```
+import '@notifi-network/notifi-react-card/dist/index.css';
+```
 
-- Configuring the card
-- Adding Event Types
-- Installing the react package into your project
-- Updating the default values to match your dapp.
+## Design Guidelines
+
+We have design recommendations on how to best present the UI to your dapp users. Check them out here: [Figma](https://www.figma.com/file/ieF0Ynuc3WI608RCt7wKSf/Notifi-Template?node-id=0%3A1&t=v8zeo6UovJAOb9vR-0).
 
+## Tutorial Video
 
-### Design Guidelines
+Here is a [link](https://www.youtube.com/watch?v=LIKu-clf9bg) covering how to setup the React Card config.
 
-We have general guidelines that we like to follow. You can see them here: [Figma](https://www.figma.com/file/ieF0Ynuc3WI608RCt7wKSf/Notifi-Template?node-id=0%3A1&t=v8zeo6UovJAOb9vR-0).
+The video covers the following:
+
+- Configuring the card in our developer tool
+- Adding Event Types
+- Installing the react package into your project
+- Updating the default values to match your dapp
 
-###### Example tips
+#### Example tips
 
 - Embed the bell icon by the wallet login section.
   ie. ![here](https://i.imgur.com/f2rnrpk.png)
+
+  &nbsp;
+
 - There should be a state for the bell icon when connected/not connected.
   ie. ![example of not connected state](https://i.imgur.com/V9yEeCj.png)
   ie. If not connected, the bell icon should be hidden.
+
 - Make sure that there is enough contrast between color selections.
+
 - Styling should be consistent with your current UI.
 
+&nbsp;
+
+# Code Examples
+
+Please see below for code examples on the component configuration. Click on the dropdown button to check out the code snippet.
+
 ### Solana
 
+<details>
```

Comment:
oh this is sick

Reply:
Thank you, Nathan! I am going to follow-up with Juni on other PR because I can't advertise Acala login works in SDK without it being in the SDK lol!